samples/msgext-northwind-inventory-csharp/Program.cs

```diff
@@ -0,0 +1,90 @@
+using msgext_northwind_inventory_csharp;
+using msgext_northwind_inventory_csharp.Bots;
+using msgext_northwind_inventory_csharp.DbSetup;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Bot.Builder.BotFramework;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using msgext_northwind_inventory_csharp.NorthwindDB;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Load configuration from appsettings.json
+builder.Configuration.AddJsonFile("appsettings.json", optional: false, reloadOnChange: true);
+
+// Register configuration in the service collection
+builder.Services.AddSingleton<IConfiguration>(builder.Configuration);
+
+// Retrieve the storage connection string from the configuration
+var storageConnectionString = builder.Configuration["StorageConnectionString"];
+
+if (string.IsNullOrEmpty(storageConnectionString))
+{
+    throw new ArgumentNullException(nameof(storageConnectionString), "Storage connection string cannot be null or empty.");
+}
+
+// Create the Bot Framework Authentication to be used with the Bot Adapter.
+var microsoftAppType = builder.Configuration["MicrosoftAppType"];
+var microsoftAppId = builder.Configuration["MicrosoftAppId"];
+var microsoftAppPassword = builder.Configuration["MicrosoftAppPassword"];
+var microsoftAppTenantId = builder.Configuration["MicrosoftAppTenantId"];
```

Comment:
from where these parameters are using?

Reply:
It was auto generated while building .net, removed it.

Reply:
ok